packages/cli/src/rollup/config.ts

```diff
@@ -188,7 +188,8 @@ runMain(command);
     onwarn(warning, rollupWarn) {
       if (
         !["CIRCULAR_DEPENDENCY", "EVAL"].includes(warning.code || "") &&
-        !warning.message.includes("Unsupported source map comment")
+        !warning.message.includes("Unsupported source map comment") &&
+        !warning.message.includes("@__PURE__")
```

Comment:
What's this for?

Reply:
Actually one library was using some invalid annotations, which rollup ignores and emits a warning every time you build. For reference check [this](https://rollupjs.org/configuration-options/#pure:~:text=Invalid%20annotations%20are%20removed%20and%20Rollup%20emits%20a%20warning.%20Valid%20annotations%20remain%20in%20the%20code%20unless%20their%20function%20call%20or%20constructor%20invocation%20is%20removed%20as%20well.)